hbase-server/src/main/java/org/apache/hadoop/hbase/monitoring/MonitoredTaskImpl.java

```diff
@@ -271,7 +270,20 @@ public void disableStatusJournal() {
 
   @Override
   public String prettyPrintJournal() {
-    return StringUtils.join("\n\t", getStatusJournal());
+    StringBuilder sb = new StringBuilder();
+    for (int i = 0; i < journal.size(); i++) {
```

Comment:
Will `journal` be null ever?

Reply:
@liuml07  That's a good point. It would thrown an NPE and thereafter abort the master if we print the journal without enabling it. Pushed the change to have a safety check before printing the journal entries

Reply:
makes sense. Thanks